addons/l10n_it_edi/models/account_move.py

```diff
@@ -889,7 +891,8 @@ def _l10n_it_edi_import_invoice(self, invoice, data, is_new):
             vat = get_text(tree, partner_info['vat_xpath'])
             codice_fiscale = get_text(tree, partner_info['codice_fiscale_xpath'])
             email = get_text(tree, '//DatiTrasmissione//Email') if partner_info['role'] == 'seller' else ''
-            if partner := self._l10n_it_edi_search_partner(company, vat, codice_fiscale, email):
+            destination_code = get_text(tree, "//CodiceDestinatario")
```

Comment:
Wait, the CodiceDestinatario is useful only when importing invoices.
If we import vendor bills... the CodiceDestinatario in the header belongs to the company receiving it (the client) so it won't work.